.travis.yml

```diff
@@ -23,6 +23,7 @@ before_install:
  # Use sed to replace the SSH URL with the public URL, then initialize submodules
  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
  - git submodule update --init
+ - git clone https://github.com/osa1/concurrent-skiplist -b refactoring ./haskell/concurrent-skiplist
```

Comment:
The reason I'm blocking this pull request is this line: I'm currently using my fork of `concurrent-skiplist`. I added this line to test travis builds. @rrnewton I have a pull request in `concurrent-skiplist`, if you merge that I can change this line and I think this is ready to merge.